Packs/CortexXDR/ReleaseNotes/2_9_1.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Palo Alto Networks Cortex XDR - Investigation and Response
+- Updated the option to filtered by the incident status
```

Comment:
Try looking at the RN templates and use them please: https://xsoar.pan.dev/docs/documentation/release-notes

Reply:
Done.